.travis.yml

```diff
@@ -4,19 +4,12 @@ cache: pip
 
 python:
   - "3.5"
-  - "2.7"
+  - "3.6"
+  - "3.7"
 
 env:
-  - DJANGO="django>=1.11,<1.12"
-  - DJANGO="django>=2.0,<2.2"
-  - DJANGO="django>=2.2,<3.0"
-
-matrix:
-  exclude:
-   - python: "2.7"
-     env: DJANGO="django>=2.0,<2.2"
-   - python: "2.7"
-     env: DJANGO="django>=2.2,<3.0"
+  - DJANGO="django>=2.1,<2.2"
+  - DJANGO="django>=2.2,<3.1"
```

Comment:
Is this intentionally `<3.1` ?

Reply:
Change to test only 2.2 and 3.0, should be:

```
  - DJANGO="django>=2.2,<3.0"
  - DJANGO="django>=3.0,<3.1"
```